### DISCUSSION TITLE
Trusted chia wallet

### DISCUSSION THREAD

**Discussion by giera4** [Chia-Network/chia-blockchain#11115] [Support]
Hellol!

I am going to buy XCH tokens and accumulate them in my wallet. Therefore, I have a question regarding the wallet. I would like it to be a wallet as an iPhone application. Do you recommend any secure application?

Maybe you have any tips on how to safely store tokens?

Thanks in advance for your help.

**Comment by giera4** [comment]
Someone help me ? :)

**Comment by loppefaaret** [comment]
We here at Chia can only really recommend our own software - that being said, there are some webwallets that you potentially could find useful ? i would recommend perhaps trying to find some stuff at reddit or googling around, and see what you find - and perhaps try and see what other people in the community says about them before jumping onboard with one
